.changelog/1423.internal.md

```diff
@@ -0,0 +1 @@
+Follow-up / clean-up after #1422
```

Comment:
If fragment text and type are the same as #1422 it will get merged in the changelog

Reply:
Oh, sweet. Did that now.